### PR TITLE
Add a readme file in automation test folder.

### PR DESCRIPTION
**Pull request by danielfenner**
Especially the "flags" directory needed a word of clarification.

Bug: b/169301133
Test: N/A, documentation only

### CODE REVIEW

File: contrib/automation_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+This directory contains end-to-end tests for Orbit. 
+
+We use pywinauto to remote control the Orbit UI. For each test there is a script $TestName.py in this folder.  
+You need a python environment to run the automation scripts. Install the dependencies:
+
+````
+pip install absl-py
+pip install pywinauto
+````
```
Comment (beckerhe) [inline contrib/automation_tests/README.md:9]: I think this should only be enclosed in 3 backticks.

**Review by beckerhe — APPROVED** [review]
Thanks a lot for the README.

I added one minor comment. Apart from that, LGTM!

File: contrib/automation_tests/README.md (hunk 1 above)
Comment (beckerhe) [reply contrib/automation_tests/README.md]: Thanks!

**Comment by danielfenner** [comment]
Thanks for having a look, fixed.

**Pull request merged**